lib/board.rb

```diff
@@ -0,0 +1,15 @@
+class Board
+
+  attr_accessor :row, :column, :grid
+# in future think about 1-D array?
+  def initialize(row, column)
```

Comment:
Why does a board need to take in two args? Isn't a board always proportional (e.g 3x3, 4x4, 5x5, etc.)?

Reply:
Good point! Changed Board to take in one argument for size.